p5js/05_FastStyleTransfer_Simple/sketch.js

```diff
@@ -0,0 +1,32 @@
+/* ===
+ML5 Example
+05_Fast_Style_Transfer_Simple
+Fast Style Transfer Simple Example with p5.js
+This uses a pre-trained model of The Great Wave off Kanagawa and Udnie (Young American Girl, The Dance)
+=== */
+
+let inputImg;
+let outputOne;
+let outputTwo;
+
+// Create two Fast Style methods with different pre-trained models
+const fs1 = new ml5.FastStyle('models/wave', modelLoaded1);
+const fs2 = new ml5.FastStyle('models/udnie', modelLoaded2);
+
+function setup() {
+  noCanvas();
+  inputImg = select('#inputImg').elt;
```

Comment:
I think I might opt for doing a p5 example with `loadImage()` and the plain JS example with an `<img>` element? This comment could apply to the `ImageNet` example and any others too. Doesn't need to be part of this pull request, we can track in a separate issue.

Reply:
I updated this so now everything is handled by p5.js